src/main/java/io/openmessaging/storage/dledger/DLedgerEntryPusher.java

```diff
@@ -427,6 +430,24 @@ private void doAppendInner(long index) throws Exception {
             lastPushCommitTimeMs = System.currentTimeMillis();
         }
 
+        private DLedgerEntry getDLedgerEntryForAppend(long index) {
+            DLedgerEntry entry;
+            try {
+                entry = dLedgerStore.get(index);
+            } catch (DLedgerException e) {
+                //  Do compare, in case the ledgerBeginIndex get refreshed.
+                if (DLedgerResponseCode.INDEX_LESS_THAN_LOCAL_BEGIN.equals(e.getCode())) {
+                    logger.info("[Push-{}]Get INDEX_LESS_THAN_LOCAL_BEGIN when requested index is {}, try to compare", peerId, index);
+                    changeState(-1, PushEntryRequest.Type.COMPARE);
+                    lastPushCommitTimeMs = System.currentTimeMillis();
```

Comment:
why update lastPushCommitTimeMs here?

Reply:
I have missed something.  This updating is useless. I'll fix this.